mentat/vision/vision_manager.py

```diff
@@ -17,12 +22,28 @@ class ScreenshotException(Exception):
 
 @attr.define
 class VisionManager:
-    driver: webdriver.Chrome | None = attr.field(default=None)
+    driver: Optional[WebDriver] = attr.field(default=None)
 
     def _open_browser(self) -> None:
         if self.driver is None or not self.driver_running():
-            service = Service(ChromeDriverManager().install())
-            self.driver = webdriver.Chrome(service=service)
+            try:
+                self.driver = webdriver.Safari()
+            except Exception:
```

Comment:
```suggestion
            except Exception as e:
                if "remote automation" in str(e).lower():
                    stream.send(f"To use Safari, enable remote automation.", color="red")
```

The first time I tried `/screenshot`, it failed (didn't recognize Safari, no backups installed). I found I needed to:

1. In the Safari menu bar, click 'Safari' and then 'Preferences'.
2. Go to the 'Advanced' tab and check 'Show Develop menu in menu bar'.
3. Open the 'Develop' menu in the menu bar and choose 'Allow Remote Automation'.

..and then it would work.

Reply:
Ah thanks. I think given that I'll try to open with chrome before telling users to enable remote automation. But if they have safari installed I'll print a different message informing them of that option.